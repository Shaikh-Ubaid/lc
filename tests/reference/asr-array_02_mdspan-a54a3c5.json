{
    "basename": "asr-array_02_mdspan-a54a3c5",
    "cmd": "lc --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/array_02_mdspan.cpp",
    "infile_hash": "0a58aca0631849d0d9f53e91255e409d79d6e8df0db2ce820f0f5d15",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-array_02_mdspan-a54a3c5.stderr",
    "stderr_hash": "475718d7cd8acad712e80ffc94ff2d8b9904c7971c3b59978ebdce38",
    "returncode": -6
}