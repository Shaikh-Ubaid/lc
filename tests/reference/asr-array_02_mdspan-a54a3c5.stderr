arr2_data is marked as read only.
